### DISCUSSION TITLE
Need help creating beautiful cards :)

### DISCUSSION THREAD

**Discussion by Sammy990** [emsesp/EMS-ESP32#1248] [Q&A]
![image](https://github.com/emsesp/EMS-ESP32/assets/39421064/d1b2a336-62c2-45d7-b0b8-e7daefe67faa)

Seeking for help on how to create this kind of beautiful card with the functionality in HA. Already have the Mqtt integration running successfully,but no experience in HA on creating additional cards (like in the attached picture)

Thanks in advance for your help

**Comment by proddy** [comment]
To start off create a blank dashboard in HA, then in HA go to Settings->Devices & Services->MQTT->Devices and click on each of the Devices that start with 'ems-esp' and then "ADD TO DASHBOARD". Do this for each of the devices and you'll end up with a dashboard with all the data automatically.

If you want to add that graph, go into the Dashboard, Edit Dashboard and add a "History Graph Card"

IF you want that button for the shower, there's some code in https://emsesp.github.io/docs/Home-Assistant/#alert-length-of-shower

**Comment by rene865** [comment]
can you show me how you made the shower duration and last shower time?

**Comment by proddy** [comment]
does https://emsesp.github.io/docs/Home-Assistant/#alert-length-of-shower help?

**Comment by rene865** [comment]
![image](https://github.com/emsesp/EMS-ESP32/assets/127300660/212d5db2-90b8-4526-a289-2af3ffc9484b)

i mean this on the dashboard. i have activated the shower timer in the EMS-ESP settings. But i can't find the sensor shower timer in home assistant.

**Reply by proddy** [reply]
I'll take a look and provide an example

**Reply by proddy** [reply]
follow https://github.com/emsesp/EMS-ESP32/issues/1267
